### ISSUE TITLE
Cannot create virtualenv with `./securedrop-admin setup` on fresh Tails 5.20 disks

### ISSUE THREAD

**Issue by nathandyer** [freedomofpress/securedrop#7084]
## Description

When trying to step through a SecureDrop installation or provisioning a new Admin or Journliast Workstation drive, the process fails at the first step while running `./securedrop-admin setup`, with a message of "Unable to create virtualenv. Check network settings and try again."

The full output is:
```
./securedrop-admin setup -v
INFO: Checking apt dependencies are installed
INFO: Setting up virtualenv
DEBUG: b'ERROR: virtualenv cannot be found.\n'
ERROR: Unable to create virtualenv. Check network settings and try again.
DEBUG: Cleaning up virtualenv
```

## Steps to Reproduce

Important note: This appears to only be happening on USB drives that were flashed with a fresh copy of Tails 5.20. I **could not** reproduce this on my primary Admin Workstation drive, with an in-place upgrade to 5.20.

1. Download Tails 5.20 .img file
2. Verify the file, then flash it to a USB drive
3. Boot the USB and create a new Persistent Storage Volume
4. Connect to Tor and clone the SD repo
5. Run `./securedrop-admin setup` from the local checkout

## Expected Behavior

A virtual environment is created and depedencies are installed.

## Actual Behavior

An error appears alerting that the virtualenv could not be created.

**Comment by nathandyer** [comment]
Trying to debug, this may or may not be useful:

`python3 -m venv admin/venv`: "The virtual environment was not created successfully because ensurepip is not available. On Debian/Ubuntu systems, you need to install the python3-venv package using the following command: `apt-get install python3-venv`

**Comment by nathandyer** [comment]
On line 211, `bootstrap.py` tries to run:

`torify virtualenv --python=python3 admin/.venv`

When trying that manually, I get:

`virtualenv: command not found`

**Comment by nathandyer** [comment]
I've discovered at least part of the underlying issue.

When you run `./securedrop-admin setup` on a fresh system, it _says_ it's checking to make sure the apt dependencies are installed, but it doesn't. It runs instantaneously, and at no point are you prompted for the local admin PW.

I discovered that if I manually run `sudo apt update` to update the local apt cache first, _then_ run `./securedrop-admin setup`, it behaves completely differently. It asks for the local admin PW, and then offers to install all the missing apt packages - which it does successfully, and then sets up the virtualenv.

It appears that a temporary workaround would be to have folks run `sudo apt update` manually first, and then try `./securedrop-admin setup` again. Of course, this doesn't explain why it's not actually checking for dependencies.
